internal/handler/wallet/backup_test.go

```diff
@@ -128,7 +128,19 @@ func Test_walletBackupAccount_Handle(t *testing.T) {
 
 		result := <-testResult
 
-		assert.Equal(t, wlt.GetPrivKey(), result.Data)
+		// Assert that the result contains the expected data structure
+		data, ok := result.Data.(struct {
+			PrivateKey string `json:"privateKey"`
+			PublicKey  string `json:"publicKey"`
+		})
+		assert.True(t, ok, "Data is not of expected type")
+
+		// Extract privateKey and publicKey from the data object
+		privateKey := data.PrivateKey
+		publicKey := data.PublicKey
+
+		assert.Equal(t, wlt.GetPrivKey(), privateKey)
+		assert.Equal(t, wlt.GetPupKey(), publicKey)
```

Comment:
no need to declare variables, and no need for the obvious chat gpt comment ;)

```suggestion
		assert.Equal(t, wlt.GetPrivKey(), data.PrivateKey)
		assert.Equal(t, wlt.GetPupKey(), data.PublicKey)
```

Reply:
always site your sources ;)